### ISSUE TITLE
Config code samples lose newlines when copied on Firefox/Windows

### ISSUE THREAD

**Issue by hyvyys** [tailwindlabs/tailwindcss.com#1192]
Works fine on Chrome:

```
module.exports = {
  theme: {
    extend: {
      spacing: {
        '5px': '5px',
      }
    }
  }
}
```

This is what I get when copying the same snippet on Firefox:

```
module.exports = {  theme: {    extend: {      spacing: {        '5px': '5px',      }    }  }}
```

(Sample from https://tailwindcss.com/docs/padding)

**Comment by thecrypticace** [comment]
Confirmed on FF for Mac as well. Strangely only an issue on Firefox. Adding newlines to the code block output fixes the copy & paste issue but adds additional new lines to the code examples. Will take some thought on how to fix this. Thanks for the report!

**Comment by stianhazel** [comment]
I think I found the problem and solution for Firefox at least.

It seems to be only affecting `language-diff-js` code blocks (the code blocks that shows the inserted lines of code). 

The normal code blocks (without the inserted lines of code) use `language-js`. In them are a bunch of `<span>`s that are direct children of the `<code>` tag.

<img width="426" alt="Screenshot 2022-02-17 at 01 42 30" src="https://user-images.githubusercontent.com/22870411/154388444-d095ecd1-edd3-47f1-8d49-33083443b779.png">

With the `language-diff-js` code blocks, each line of code is separated into its own `<span>` which indicates if it's highlighted or not with the `inserted` and `unchanged` classes

<img width="492" alt="Screenshot 2022-02-17 at 01 47 18" src="https://user-images.githubusercontent.com/22870411/154388789-edb910c1-65b2-4c26-81fa-d79efa0c6137.png">

Removing the container `<span>` tags resolves the issue, but obviously you'd lose the `inserted` and `unchanged` classes.

So I tried changing the `<span>` to a `<div>` and that resolves the issue, `<span>` seems to break the newlines on Firefox, despite them having `display: block` in the CSS.

<img width="495" alt="Screenshot 2022-02-17 at 01 51 20" src="https://user-images.githubusercontent.com/22870411/154389225-52116fd8-30c6-4548-8596-d1b6a6f8988d.png">

I changed the top 3 into `<div>` (the first 3 lines) and they retain the new lines for those 3.

<img width="605" alt="Screenshot 2022-02-17 at 01 52 29" src="https://user-images.githubusercontent.com/22870411/154389281-6e0d25b4-1544-48bc-a4f2-e6d6ca788cb7.png">
